comics/comics/ikkesaro.py

```diff
@@ -0,0 +1,18 @@
+# encoding: utf-8
```

Comment:
Thanks for explaining this. I was wondering why some crawlers had it and someone not.